comparison/ansible/roles/retr0h.etcd/.travis.yml

```diff
@@ -7,7 +7,7 @@ services:
 
 before_install:
   - sudo apt-get -qq update
-  - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
+  - sudo apt-get --no-install-recommends install apt-utils ca-certificates -o Dpkg::Options::="--force-confold" --force-yes docker-engine
```

Comment:
I am not fully confident in only using `--force-yes`. It might be good to keep the `-y` here as well.